e2e_test/shared.sh

```diff
@@ -9,6 +9,14 @@ case $NETWORK in
     # for contract in GoldToken FeeHandler FeeCurrencyDirectory
     #      cast call 0x000000000000000000000000000000000000ce10 "getAddressForStringOrDie(string calldata identifier) returns (address)" $contract
     #  end
+    mainnet)
+      export ETH_RPC_URL=https://forno.celo.org/cel2
```

Comment:
```suggestion
      export ETH_RPC_URL=https://forno.celo.org
```

Reply:
I did look at the address to check if is already, but apparently the `/cel2` was too subtle for me to notice (or I was too blind to see it). Fixed